network-api/networkapi/wagtailpages/pagemodels/products.py

```diff
@@ -390,17 +393,39 @@ class ProductPage(FoundationMetadataPageMixin, Page):
         ),
     ]
 
+    @property
+    def product_type(self):
+        if isinstance(self, SoftwareProductPage):
+            return "software"
+        elif isinstance(self, GeneralProductPage):
+            return "general"
+        else:
+            return "standard"
+
+    def get_context(self, request, *args, **kwargs):
+        context = super().get_context(request, *args, **kwargs)
+        context['product'] = self
+        context['categories'] = BuyersGuideProductCategory.objects.all()
+        context['mediaUrl'] = settings.CLOUDINARY_URL if settings.USE_CLOUDINARY else settings.MEDIA_URL
+        context['coralTalkServerUrl'] = settings.CORAL_TALK_SERVER_URL
+        context['pageTitle'] = pgettext(
+                'This can be localized. This is a reference to the “*batteries not included” mention on toys.',
+                '*privacy not included') + f' - {self.title}'
+        return context
+
     class Meta:
         verbose_name = "Product Page"
 
 
 class SoftwareProductPage(ProductPage):
+    template = 'buyersguide/product_page.html'
```

Comment:
is the `template` value not inherited?

Reply:
Seemingly no, it's not. Had to add this back into the child page classes.